tests/test5.yaml

```
@@ -478,6 +497,27 @@ sensor:
     max_flow_rate:
       name: Max Flow Rate
 
+  - platform: vbus
+    model: deltasol c
+    temperature_3:
+      name: Temperature 3
+    operating_hours_1:
+      name: Operating Hours 1
+    heat_quantity:
+      name: Heat Quantity
+    time:
+      name: System Time
+
+  - platform: vbus
+    model: custom
+    command: 0x100
+    source: 0x1234
+    dest: 0x10
+    sensors:
+      - id: vcustom
+        name: VBus Custom Sensor
+        lambda: return x[0] / 10.0
```

CI error due to missing semicolon
```suggestion
        lambda: return x[0] / 10.0;
```